jwst/assign_wcs/assign_wcs.py

```diff
@@ -94,5 +95,10 @@ def load_wcs(input_model, reference_files={}, nrs_slit_y_range=None):
                     log.info("Unable to update S_REGION for type {}: {}".format(
                         output_model.meta.exposure.type, exc))
 
+        # Store position of dithered pointing location in metadata for later spectral extraction
+        if output_model.meta.exposure.type.lower() == 'mir_lrs-fixedslit':
+            output_model = store_dithered_position(output_model)
```

Comment:
Since the function doesn't have a return value this should be changed too, otherwise `output_model` is  None.

Reply:
🤦